project/Dependencies.scala

```diff
@@ -362,10 +364,12 @@ object Dependencies {
 
   val Sqs = Seq(
     libraryDependencies ++= Seq(
-      "com.amazonaws" % "aws-java-sdk-sqs" % AwsSdkVersion, // ApacheV2
-      "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion % Test, // ApacheV2
-      "org.mockito" % "mockito-core" % "2.23.4" % Test // MIT
-    )
+      "com.github.matsluni" %% "aws-spi-akka-http" % "0.0.4" excludeAll ExclusionRule(organization = "com.typesafe.akka"), // ApacheV2
+      "software.amazon.awssdk" % "sqs" % AwsSdk2Version excludeAll(ExclusionRule(organization = "software.amazon.awssdk", name = "netty-nio-client"), ExclusionRule(organization = "io.netty")), // ApacheV2
+      "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion, // ApacheV2
+      "org.mockito" % "mockito-core" % "2.23.4" % Test, // MIT
+      "org.mockito" % "mockito-inline" % "2.23.4" % Test // MIT
+  )
```

Comment:
Formatting of this closing parenthesis seems off.

Reply:
fixed